Dockerfile-development

```diff
@@ -25,6 +25,8 @@ COPY Makefile ${USER_HOME}
 WORKDIR ${USER_HOME}
 
 RUN make dependencies
+RUN mv lua_modules /tmp/lua_modules
+
 RUN cpanm --installdeps ./gateway
 
 COPY . ${USER_HOME}
```

Comment:
We could add `lua_modules` to `.dockerignore` so it is not copied?

Reply:
Yes, it's because of the volume.